examples/vault_agent.py

```diff
@@ -2,51 +2,23 @@
 import os
 
 from dotenv import load_dotenv
-from requests.exceptions import ConnectionError
 
 from notte.agents.falco.agent import FalcoAgent as Agent
 from notte.agents.falco.agent import FalcoAgentConfig as AgentConfig
 from notte.common.credential_vault.hashicorp.vault import HashiCorpVault
 
-# Load environment variables
+# Load environment variables and create vault
```

Comment:
add a comment to explain to users that they have to setup:
VAULT_URL
VAULT_DEV_ROOT_TOKEN_ID in the .env